addons/web/static/src/views/list/list_renderer.xml

```diff
@@ -228,7 +228,7 @@
                         t-att-class="getCellClass(column, record)"
                         t-att-name="column.name"
                         t-att-colspan="column.colspan"
-                        t-att-data-tooltip="getCellTitle(column, record)"
+                        t-att-data-tooltip="!evalModifier(column.modifiers.invisible, record) ? getCellTitle(column, record) : false"
```

Comment:
```suggestion
<t t-set="isInvisible" t-value="evalModifier(column.modifiers.invisible, record)"/>
                        t-att-data-tooltip="!isInvisible ? getCellTitle(column, record) : false"
```

Reply:
with the t-set above the td obviously

then isInvisible can be used below as well